java/src/main/java/com/marginallyclever/filters/Filter_GeneratorYourMessageHere.java

```diff
@@ -64,10 +67,9 @@ protected void createMessage(String str,String dest) {
 			textSetAlign(Align.RIGHT);
 			textSetVAlign(VAlign.TOP);
 			textSetPosition(image_width,image_height);
-			textCreateMessageNow("Makelangelo #"+Long.toString(machine.getUID()),output);
-			
-			output.close();
```

Comment:
Hey, it's all good.  You do great work.  Even the best have slip ups.

On Wed, Jun 24, 2015 at 9:54 AM, Peter Colapietro <notifications@github.com>
wrote:

> In
> java/src/main/java/com/marginallyclever/filters/Filter_GeneratorYourMessageHere.java
> <https://github.com/MarginallyClever/Makelangelo/pull/117#discussion_r33169436>
> :
>
> > @@ -64,10 +67,9 @@ protected void createMessage(String str,String dest) {
> >  			textSetAlign(Align.RIGHT);
> >  			textSetVAlign(VAlign.TOP);
> >  			textSetPosition(image_width,image_height);
> > -			textCreateMessageNow("Makelangelo #"+Long.toString(machine.getUID()),output);
> > -			
> > -			output.close();
>
> Sorry about removing this. Not sure why this revision made it into this
> branch.
>
> I do have a branch which uses try with resources
> <https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html>
> making the explicit close of streams superfluous but as this is not that
> branch, I do not know why this was removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MarginallyClever/Makelangelo/pull/117/files#r33169436>
> .
>



-- 
Dan Royer :: Marginally Clever <http://www.marginallyclevergroup.com/> ::
Raising Robot Literacy :: +1.604.916.2281